README.md

```diff
@@ -33,15 +33,51 @@ Each markdown file will be modified with a dox header, and `.dox.yaml` will be
 updated to include the page_id of the generated root page. Be sure to commit
 `.dox.yaml` and these changes to markdown in your source code management.
 
-### Ignored Files
+## File Directives
 
-If a file should not be published, add the ignore directive to the top of the file:
+dox allows the user to control how files are published through file directives.
+The directives are defined in the dox header, which is a single line comment at
+the top of source files. The header contains a comma separated list **(no white
+space)** of directives. They have the following format:
+
+```
+<!-- dox: <directive>,<directive>,<directive> -->
+```
+
+All published pages will have the page ID included in the dox header.
+
+```
+<!-- dox: <directive>,1234567890 -->
+```
```

Comment:
I really like this direction. Should we refer to these as "Source Directives", and refer to "files" as "source files" in this section? Aesthetically, I'd like to see the source ID at the beginning of the directive list. We might also consider that some future source directives might be some value other than a boolean switch. But keeping with the problem at hand, could we could require that the first directive be either `ignore` or the page ID, optionally followed by any other directives?